packages/synapse-client/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "synapse-client",
```

Comment:
`synapse-client` or `@sage-bionetworks/synapse-client`?

Reply:
Seems like we have adopted @sage-bionetworks/name based on packages like @sage-bionetworks/synapse-types.   What do you think?

Reply:
Works for me! I'll make the change in a follow-up PR, since this isn't going to be published yet.